library/core/src/num/uint_macros.rs

```diff
@@ -2704,12 +2704,26 @@ macro_rules! uint_impl {
         #[rustc_const_unstable(feature = "isqrt", issue = "116226")]
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
-        #[inline]
+        #[inline(always)]
```

Comment:
```suggestion
        #[inline(always)] // Inline the wrapper, the inner routine can outline
```

To be consistent with https://github.com/rust-lang/rust/pull/128166/files#r1722390704

Reply:
Okay yeah @ChaiTRex if you change from `#[inline(always)]` to `#[inline]` you can forget about these comments